tests/reports/tests-reports-functions.php

```diff
@@ -792,7 +792,7 @@ public function test_clear_filter_should_default_to_last_30_days() {
 
 	public function test_gross_order_status() {
 		$expected = array(
-			'completed',
+			'complete',
```

Comment:
I know you asked about this in Slack, so I'm just noting here to make doubly sure `complete` is correct. 📝